doc/releases/release-notes-2.1.rst

```diff
@@ -91,6 +93,13 @@ Drivers and Sensors
   * Added Atmel AT25 (and compatible) SPI EEPROM driver
   * Added native_posix EEPROM emulation driver
 
+* CAN
```

Comment:
Ah, I know i am too picky here, but is it possible to put CAN section before EEPROM?, so it is in alphabetical order

Reply:
No problem!